### ISSUE TITLE
Graph Api Bad Signature IOS

### ISSUE THREAD

**Issue by SnnYsf** [facebook/facebook-sdk-for-unity#723]
## Checklist

Unity SDK 17.0.2

https://graph.facebook.com/{id}/picture

If limited login is set in Facebook settings, the guest silhouette always returns.
If you switch to advanced permissions, the user image appears.

If you use access token with limited permissions, the user image will appear on Android. However, since limited login is set in iOS, the guest silhouette appears in the graph API call. If you add access token, you will get a bad signature error.

In short, it seems impossible to get a 200*200 photo of a user in iOS limited login.
